intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -2298,6 +2298,66 @@ def convert_date(value):
     }
 }
 
+# https://www.shadowserver.org/what-we-do/network-reporting/accessible-radmin-report/
+accessible_radmin = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port', convert_int),
+    ],
+    'optional_fields': [
```

Comment:
I'm missing the `version` field here. Value is mostly `Radmin v3.X Radmin Authentication`

Reply:
of course, thanks